parallel-install/pkg/deployment/deployment.go

```diff
@@ -27,20 +33,24 @@ type Deployment struct {
 	ResourcesPath string
 	Cfg           config.Config
 	// Used to send progress events of a running install/uninstall process
-	ProcessUpdates chan<- ProcessUpdate
+	ProcessUpdates   chan<- ProcessUpdate
+	kubeClient       kubernetes.Interface
+	metadataProvider metadata.MetadataProvider
 }
 
 type Installer interface {
```

Comment:
Do we really need that interface any longer as we have only one implementation of it?

Reply:
we agreed to get rid of this interface